README.md

```diff
@@ -12,8 +12,46 @@
 [![Stability: unstable](https://img.shields.io/badge/stability-unstable-yellowgreen.svg)](https://nodejs.org/api/documentation.html#documentation_stability_index)
 [![Join the chat at https://gitter.im/1-liners/1-liners](https://img.shields.io/badge/gitter-join%20chat-1dce73.svg)](https://gitter.im/1-liners/1-liners)
 
-> Useful oneliners and shorthand functions
-
+<sup>MADE IN SWITZERLAND</sup>
+
+1-liners
+====
+
+**Functional tools that couldn’t be simpler.**
+
+We’re proud to present *1-liners* – a stupid-simple functional utility belt. 62 *[@stoeffel this can update automatically]* one-liner functions, hand-crafted with love and attention.
```

Comment:
Yeah. Switzerland, India, Poland and Germany to be exact :)

It just sounds good that way :) It reads: “This is a quality product developed with obsessive attention to detail.”